ePBFT/src/pbft/libbyz/Replica.cpp

```diff
@@ -304,8 +310,11 @@ bool Replica::apply_ledger_data(const std::vector<uint8_t>& data)
     else
     {
       LOG_DEBUG << "Received entries could not be processed. Received seqno: "
```

Comment:
Minor: clang-format on this line seems a bit off?

Reply:
The formatting check seems to pass so I don't know.